docs/intro/install.rst

```diff
@@ -118,6 +118,27 @@ Once you've installed `Anaconda`_ or `Miniconda`_, install Scrapy with::
 
   conda install -c conda-forge scrapy
 
+To install Scrapy on Windows using ``pip`` :-
+
+1. Install latest version of `Python`_ (Recommended to tick [Add Python to PATH] checkbox during installation)
+
+2. Install or upgrade pip::
+
+    python -m pip install --upgrade pip
+
+3. Create Venv (Virtual Environment)::
+
+    python -m venv [env name]
+
+4. Download `Microsoft Visual C++ 14.0`_
+
+5. Run the downloaded EXE file and let it setup the Visual Studio Installer
+
+6. Open Visual Studio Installer, select and install C++ build tools
```

Comment:
I’m a bit confused about these 3 steps (4, 5, 6).

I think it may be best to refactor them a bit to make clear what you are installing. For example:

1. Download and execute the **Microsoft C++ Build Tools** to install the **Visual Studio Installer**.
2. Open the **Visual Studio Installer**, and select and install the **C++ build tools**.

But I think I might be misunderstanding how this works. https://stackoverflow.com/a/55053709/939364 suggests that the exe opens an installer, and that you need to select 2 packages to be installed with it. If that is so, these steps need to be refactored.

All in all, if https://stackoverflow.com/a/55053709/939364 is correct, I think we don’t even need a list. A paragraph can explain what https://stackoverflow.com/a/55053709/939364 describes, making sure we use the component names as they appear in the installer. Then a second paragraph can link back to “Installing Scrapy” as I suggested above, which covers installing Scrapy and virtual environments.

Reply:
To make the documentation more future proof, I would omit versions (e.g. MSCV, Windows SDK); if it seems too ambiguous, maybe provide version-specific names as examples.

As for the size warning, does it really need to download 4 GiB if you only enable those 2 options? If so, I’m OK with including a warning in our documentation, but I’m worried about indicating a specific size in our documentation and that changing upstream with time. I’m not sure how to word it without a specific size or range, though. Maybe warn that this installation method requires significantly more disk space than Anaconda? (assuming that’s true).

Reply:
> To make the documentation more future proof, I would omit versions (e.g. MSCV, Windows SDK); if it seems too ambiguous, maybe provide version-specific names as examples.

Alright I will omit versions and add an example to makes things less ambiguous.